change/@ni-nimble-components-31e75753-f86e-4adf-8800-8c1ea12d516b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Moving to threaded comment:
@atmgrifter00 
So, I've noticed in the [Combobox Storybook Doc page](https://60e89457a987cf003efc0a5b-jfqgyfrjkn.chromatic.com/?path=/docs/combobox--docs) that the dropdown is still clipped within its parent container. Were we not expecting this to be fixed with this change?

Reply:
looks like in chrome it pops over but firefox it's still stuck with the same storybook

Reply:
I've run into enough problems and uncertainties that I'm planning on using anchored region for the select and combobox, instead. When that is ready, I'll close this PR and open a new one.